src/rpc/rawtransaction.cpp

```diff
@@ -2048,10 +2048,12 @@ RPCHelpMan descriptorprocesspsbt()
         sighash_type,
         finalize);
 
-    // Check whether or not all of the inputs are now signed
+    // Check whether or not all of the inputs are now correctly signed 
```

Comment:
In 7e468a6fb01b36edb6955ccb2b869ec1063a891c "rpc: Fix descriptorprocesspsbt internal bug on invalid signatures"

Trailing whitespace on this line